<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandscapeWidget</class>
 <widget class="QWidget" name="LandscapeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Turn on sculpting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>MinX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>MaxX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>MinZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>MaxZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>h</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSpinBox" name="minXSpinBox">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="value">
            <number>-60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxXSpinBox">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="minZSpinBox">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="value">
            <number>-60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxZSpinBox">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>2.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Construct</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sensitivity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="SensitivitySpinBox">
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="RadiusSpinBox">
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
